streampipes-extensions/streampipes-processors-filters-jvm/src/main/resources/org.apache.streampipes.processors.filters.jvm.sdt/strings.en

```diff
@@ -0,0 +1,34 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+org.apache.streampipes.processors.filters.jvm.sdt.title = Swinging Door Trending (SDT) Filter Processor
+org.apache.streampipes.processors.filters.jvm.sdt.description = The Swinging Door Trending (SDT) Filter Processor can extract and forward the characteristic events of the original stream. In general, this filter can also be used to reduce the frequency of original data in a lossy way.
+
+sdt-timestamp-field.title = Timestamp Field
+sdt-timestamp-field.description = Specifies the timestamp field name where the SDT algorithm should be applied on.
+
+sdt-value-field.title = Value Field
+sdt-value-field.description = Specifies the value field name where the SDT algorithm should be applied on.
+
+sdt-compression-deviation.title = Compression Deviation
+sdt-compression-deviation.description = Compression Deviation is the most important parameter in SDT that represents the maximum difference between the current sample and the current linear trend. Compression Deviation needs to be greater than 0 to perform compression.
+
+sdt-compression-min-interval.title = Compression Minimum Time Interval
+sdt-compression-min-interval.description = Compression Minimum Time Interval is a parameter measures the time distance between two stored data points, which is used for noisy reduction. If the time interval between the current point and the last stored point is less than or equal to its value, current point will NOT be stored regardless of compression deviation. The default value is 0 with time unit ms.
+
+sdt-compression-max-interval.title = Compression Maximum Time Interval
+sdt-compression-max-interval.description = Compression Maximum Time Interval is a parameter measure the time distance between two stored data points. If the time interval between the current point and the last stored point is greater than or equal to its value, current point will be stored regardless of compression deviation. The default value is 9,223,372,036,854,775,807(Long.MAX_VALUE) with time unit ms.
```

Comment:
Since these descriptions are displayed in the UI when configuring the processing element, they are probably too long to be rendered properly.
Although, I really appreciate your take in providing a solid documentation 💪🏼

Reply:
Yes, I actually noticed this potential UI rendering problem.

<img width="709" alt="image" src="https://user-images.githubusercontent.com/30497621/219767694-3595cf25-49b0-4edc-86ea-16bd7ec71a8a.png">

I checked the rendering on my computer and found that there was no problem. But I'm not sure if there will be rendering problems on other computer environments.

Would I need to reduce some text content from here? 😂

Reply:
Although, I really like your detailed explanation I would suggest to summarize it in one or two short sentences.
We still have an extensive description in the documentation file :)

Reply:
I really like the detailed description. I'll have a look at the UI to see if we can improve the presentation of long descriptions.

Reply:
@dominikriemer would it be possible to have a collapsible UI element? Then we could provide a short and a long description in the `strings` file
Per default the short description would be displayed but one can get the extensive one by uncollapsing the element

Reply:
The way now the long description display actually is good to me. But yes, the collapsible UI idea sounds better :)